js/index.js

```diff
@@ -1,4 +1,21 @@
 var environment = require('./environment')
+var apod = require('nasa-apod')
+var $ = require('jquery')
+
+var randomDate = function randomDate(start, end) {
+    return new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime()));
+}
+
+var date = randomDate(new Date(2012, 0, 1), new Date())
+
+apod(date).then(function(data) {
+    $('#body').css("background-image","url("+data.hdurl+")")
```

Comment:
there'll only be on `<body>` in `index.html` so we can omit the ID, and do something like

```javascript
var $body = $('body')
$body.css('background-image', 'url(' + data.hdurl + ')'
...
...
```